includes/admin/class-wc-admin-duplicate-product.php

```diff
@@ -43,6 +43,18 @@ public function dupe_link( $actions, $post ) {
 			return $actions;
 		}
 
+		// Add Class to Delete Permanently link in row actions.
+		$total_sales = get_post_meta( $post->ID, 'total_sales', true );
+		if( 'publish' === $post->post_status && 0 < $total_sales ) {
```

Comment:
Why is trash disabled if there are no sales, rather than giving the choice?

Reply:
@mikejolley I've not disabled Trash when there is no sales. Trash will work as it is but the confirmation popup will not appear if there is no sales. Let me know if there is any change to the way of implementation.